src/components/policy/policy_external/test/sql_pt_representation_test.cc

```diff
@@ -62,10 +62,15 @@ using testing::ReturnRef;
 using testing::Return;
 using testing::NiceMock;
 using testing::Mock;
+
 namespace test {
 namespace components {
 namespace policy_test {
 
+namespace {
+const int32_t kPolicyTablesNumber = 27;
```

Comment:
@LuxoftAKutsan The answer for your question could be found by taking a look at the related change in RefreshDB_DropExistedPTThenRefreshDB_ExpectTablesWithInitialData test.